core/injector/application_injector.cpp

```diff
@@ -868,6 +868,7 @@ namespace {
             di::bind<crypto::SessionKeys>.template to<crypto::SessionKeysImpl>(),
             di::bind<network::SyncProtocol>.template to<network::SyncProtocolImpl>(),
             di::bind<network::StateProtocol>.template to<network::StateProtocolImpl>(),
+            di::bind<network::IBeefyProtocol>.template to<network::BeefyProtocol>(),
```

Comment:
Let's use de-facto applied naming convention: Name/NameBase for interfaces and NameImpl for implementation